snmp/README.md

```diff
@@ -4,211 +4,102 @@
+| `discovery_allowed_failures` | The number of times a discovered host can fail before being removed from the list of discovered devices.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `bulk_threshold`             | The number of symbols in a table that triggers a BULK request. This paramater is only relevant for SNMPv > 1.                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `tags`                       | A list of global tags to add to all SNMP metrics. Read more about [tagging in Datadog][6].                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 
-    ## @param snmp_version - integer - optional - default: 2
-    # snmp_version: 2
-
-    ## @param metrics - list of elements - (required if none at the init_config level)
-    metrics:
-    - MIB: <MIB_NAME>
-      symbol: <SYMBOL>
-    - OID: <OID>
-      name: <OID_NAME>
-      metric_tags:
-        - <TAG>
-```
-
-##### SNMP v3 configuration
-
-**Note**: See the [SNMP Library reference][4] for all configuration options.
+##### Sample config
 
 ```yaml
 init_config:
-  ## @param mibs_folder - string - required
-  # mibs_folder: <PATH_TO_ADDITIONAL_MIBS>
-
-  ## @param global_metrics - list of elements - (required if none at the instance level)
-  # global_metrics:
-  ## - MIB: <MIB_NAME>
-  ##   symbol: <SYMBOL>
+  profiles:
+    f5-big-ip:
+      definition_file: f5-big-ip.yaml
+    router:
+      definition_file: generic-router.yaml
 
 instances:
-    ## @param ip_address - string - required
-  - ip_address: localhost
+   ## @param network_address - string - optional
+   -  
```

Comment:
Some formatting suggestions based on the [rendered view](https://github.com/DataDog/integrations-core/blob/68ca528ab136b3bbcf9805efd6e2e556d941f41f/snmp/README.md#sample-config): I think we need to move the delimiting `-` at the top, and then align comments with their respective parameters...

```diff
-   ## @param ...
-   -
+  -
+    ## @param ...
     network_address: ...

-   ## @param ...
+    ## @param ...
     port: 161
```

Reply:
updated in most recent commit